pkg/vulnsrc/osv/osv.go

```diff
@@ -0,0 +1,223 @@
+package osv
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"path/filepath"
+	"strings"
+	"time"
+
+	bolt "go.etcd.io/bbolt"
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/trivy-db/pkg/db"
+	"github.com/aquasecurity/trivy-db/pkg/types"
+	"github.com/aquasecurity/trivy-db/pkg/utils"
+	"github.com/aquasecurity/trivy-db/pkg/vulnsrc/vulnerability"
+	vtypes "github.com/aquasecurity/vuln-list-update/osv"
+)
+
+const (
+	osvDir           = "osv"
+	datasourceFormat = "osv-%s"
+	platformFormat   = "Osv Security Advisories %s"
+
+	//ecosystem names
+	Python = "PyPI"
+	Go     = "Go"
+	Rust   = "crates.io"
+)
+
+var defaultEcosystem = []ecosystem{
```

Comment:
consider using `map` here this way `getEcoSystem()` won't be required

Reply:
also variable name should be in plural form

Reply:
made `map`, remove `getEcoSystem()`